components/content-blocks/Callout/CalloutEntry/styles.js

```diff
@@ -0,0 +1,206 @@
+import styled from "styled-components";
+import { MixedLink } from "@rubin-epo/epo-react-lib";
+import SharePopupComponent from "@/layout/SharePopup";
+import {
+  fluidScale,
+  containerRegular,
+  respond,
+  tokens,
+} from "@/styles/globalStyles";
+import { Section as CalloutSection } from "../styles";
+
+export const Section = styled(CalloutSection)`
+  position: relative;
+`;
+
+export const Inner = styled(MixedLink)`
```

Comment:
This `Section` is identical to the `Section` in `/content-blocks/Callout`, except for the positioning.  This felt like the styled componenty way to accomplish this, but maybe you have a better pattern?  `Inner` was similar to the `Inner` in `/content-blocks/Callout`, but then departed enough from it that I felt like it made sense to have it be its own thing (i.e. not extended from the other `Inner`).